docs/past_meetups/posts/2023-09-11-josh.md

```diff
@@ -6,8 +6,9 @@ authors:
     - josh-schneider
 ---
 
-# Writing Code Other People Have to Maintain
-Lorem Ipsum
+# Best Practices for Writing Code for Other People Have to Use
+
+Josh has opinions!
```

Comment:
Can we add a little more here? Like a one sentence topic sentence on what you're going to cover? Don't worry too much about below the fold, we'll update that after the presentation.

Reply:
I merged the teaser sentence with the content below. Let me know what you think of that, or if you are looking for even more detail. @bradsbrown , let me know what you think too.

Reply:
I think that with it all pulled together it looks solid to me. @MasonEgger's the one with write access though! 😂